HertzWatch WatchKit Extension/ContentView.swift

```diff
@@ -146,6 +136,18 @@ struct ContentView: View {
                 }
             }
         }
+        .gesture(DragGesture()
+                            .onChanged { gesture in
+                                    self.offset = gesture.translation
+                            }
+                            .onEnded { gesture in
+                                if abs(self.offset.height) > 20 {
+                                    InstructionView()
```

Comment:
So here's your issue, you got bugs in the carbonator. As you noticed, calling a InstructionView from here won't work for the simple reason that it's a view. What you need to do is add another @State variable and then in the main view code, that is in the same way we check for example if the model.isRunning you would do the same thing with the new State variable.